app/Actions/Dislike/DislikePlaceAction.php

```diff
@@ -51,7 +52,23 @@ public function execute(DislikePlaceRequest $request): DislikePlaceResponse
         if (!$place) {
             throw new PlaceNotFoundException;
         }
+
+        $liked = LikeStatus::none();
+        $userId = Auth::id();
+        $like = $this->likeRepository->findByUserAndPlace($userId, $request->getPlaceId());
+        if ($like) {
+            $liked = LikeStatus::liked();
+        } else {
+            $dislike = $this->dislikeRepository->findByUserAndPlace($userId, $request->getPlaceId());
```

Comment:
exists above
`$dislike = $this->dislikeRepository->findByUserAndPlace($userId, $request->getPlaceId());`